lemur/certificates/views.py

```diff
@@ -128,10 +131,15 @@ def get(self):
            :statuscode 403: unauthenticated
 
         """
-        parser = paginated_parser.copy()
```

Comment:
The paginated_parser was putting default values for sorting and paging. It was not used eventually in the old code. As commented below, using non-paginated parser to ensure backward compatibility